src/extra/components/scan.py

```diff
@@ -339,10 +348,11 @@ def _get_motor_steps(self, actual_pos, resolution, min_trains, intra_step_filter
         # Detect backlash at the beginning by comparing the step direction of
         # the first and second steps. If there's backlash they're typically not
         # the same, so we remove the first step if so.
-        first_step = steps[1][0] - steps[0][0]
-        second_step = steps[2][0] - steps[1][0]
-        if np.sign(first_step) != np.sign(second_step):
-            del steps[0]
+        if len(steps) > 3:
```

Comment:
Should this be `>=`? It looks like we only need 3 values to do the check. I guess most real scans will have quite a few more steps in any case.

Reply:
Ah yes, good catch. Fixed in 73fccbd.